Automattic/jetpack
# Protect: Add global WAF stats

## Description
Adds global WAF statistics to the Protect initial state.

Reliant on `D155606`
 
## Proposed changes:

* Retrieves global stats from the new wpcom endpoint
* Displays them where applicable

### Other information:

## Jetpack product discussion


- https://github.com/Automattic/jetpack-scan-team/issues/1286

## Does this pull request change what data or activity we track or use?


- No

## Testing instructions:






* Checkout this branch and start up your Jurassic Tube
* If `D155606` is not yet in production, checkout that branch on your `wpsandbox` and update your 0-sandbox file in your local jetpack repo to define `JETPACK__SANDBOX_DOMAIN`  accordingly to connect up
* Review the initial state and verify that all globalStats are present
* Review the sections where the new total vulns data displays and ensure it is an accurate and consistent value

## Screenshots:

Scanning:
![Screen Shot on 2024-07-17 at 12-57-39](https://github.com/user-attachments/assets/3c3982ed-6077-40b0-bf34-729e52a62f7c)

Footer:
![Screen Shot 2024-07-17 at 13 01 42](https://github.com/user-attachments/assets/9e9a541f-94df-4635-9116-0855e251942e)
